src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatRequest {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
```

Comment:
Using generic wildcard '?' in List<?> messages may result in type safety issues and make it difficult to maintain the code. It is advised to specify the actual type of objects that will be stored in the list.